src/styles/_breadcrump.scss

```diff
@@ -0,0 +1,46 @@
+@use 'global';
+
+.breadcrump {
+    display: flex;
+    justify-content: right;
+    flex-grow: 1;
+    width: 100%;
+    padding: 2rem 1rem;
+    &__list {
+        list-style: none;
+        display: flex;
+    }
+}
+
+.breadcrump-item {
```

Comment:
Can breakcrump-item be used by itself without breadcrumb? Yes -> It's a block. No -> It's an element

Reply:
again i needed to create a new block context so i could define elements for the different bread crumb components although now thinking about it this actually is wrong